roles/config-rh-sso/tasks/add_role.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+- name: "Requesting Access Token"
+  uri:
+    url: "{{ rh_sso_protocol }}://{{ rh_sso_host }}:{{ rh_sso_port }}/auth/realms/master/protocol/openid-connect/token"
+    method: POST
+    body_format: form-urlencoded
+    body:
+      username: "{{ sso_admin_user }}"
+      password: "{{ sso_admin_pass }}"
+      grant_type: "password"
+      client_id: "admin-cli"
+    validate_certs: no
+  register: rh_sso_token
+
+- name: "Retrieve template for role"
+  set_fact:
+    role_json: "{{ lookup('template', 'realm_roles.json.j2') | to_json }}"
+
+- name: "Create Role for service Red Hat Single Sign-On"
+  uri:
+    url: "{{ rh_sso_protocol }}://{{ rh_sso_host }}:{{ rh_sso_port }}/auth/admin/realms/master/roles"
+    method: POST
+    body: "{{ role_json }}"
+    body_format: json
+    status_code: 201, 409
```

Comment:
I think long term we need to rethink this. We don't  want to be okay on a role create when the response is 409 because that just says there is a conflict and skips it (i.e. wouldn't get any of the updates). Let's just create an issue to follow up on this though.

Reply:
This also applies to a handful of our other roles that we've created at this point.

Reply:
Makes sense. I have created the issue #499 for this.